src/View/Components/Avatar.php

```diff
@@ -84,6 +84,8 @@ public function personalization(): array
                     ],
                 ],
             ],
+            'border' => 'border-2',
```

Comment:
Transform this into `border`:

```suggestion
            'border' => [
                'base' => 'border-2',
                'radius' => 'rounded-full',
            ],
```